Cargo.lock

```diff
@@ -147,7 +147,7 @@ version = "0.7.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "c5f1ee878fc434a52b8fc258418a9b1c08273c48599da652cdd79c43d6ad9f10"
 dependencies = [
- "clap",
+ "clap 2.34.0",
```

Comment:
We should've done this a long time ago, but we should disable the `binary` feature in `av-scenechange` to avoid having it pull in clap 2. We can do this with just `default-features = false` in the dependency line for it.

Reply:
Thanks! Disabled the default features for av-scenechange.